tracer/test/snapshots/AspNetWebApi2Tests.VirtualApp.Integrated.WithFF.__path=_api2_statuscode_201-ps=false&ts=false_statusCode=500.verified.txt

```diff
@@ -0,0 +1,47 @@
+[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: CustomTracingExceptionHandler.handle-async,
+    Resource: CustomTracingExceptionHandler.handle-async,
+    Service: sample/my-app,
+    ParentId: Id_3,
+    Tags: {
+      env: integration_tests,
+      language: dotnet,
+      span.kind: server,
+      version: 1.0.0
+    }
+  },
+  {
+    TraceId: Id_1,
+    SpanId: Id_3,
+    Name: aspnet.request,
+    Resource: GET /my-app/api2/statuscode/?,
```

Comment:
FYI this is one of the examples where not running the WebApi integration results in the resource name (incorrectly) including the site path `/my-app`. This is being addressed in #2852